src/components/Stepper/Step/Step.scss

```diff
@@ -0,0 +1,113 @@
+@import "vanilla-framework";
+
+.p-inline-list__item {
+  margin: 0;
+}
```

Comment:
This is overriding a vanilla class on the root level. It should be namespaced.

Reply:
Good catch - addressed the [changes](https://github.com/canonical/react-components/compare/ea71adb0101ff639008ef938152a337c3bf734bc..00fb6effdfa8ff1b63b7af74f3ee26d201ce47c7)